cl21.md

```diff
@@ -2,6 +2,12 @@
 title: Common Lisp for the 21st century
 ---
 
+CAUTION: We were excited by `CL21`, but we must admit that the project
+is staling and suffers from unresolved issues. We can't recommend it
+anymore. Hopefully, you'll now find many equivalent features in other
+libraries (`generic-cl`, `rutils`, `access` and many more).
+
+
 [CL21](http://cl21.org/) is an experimental project redesigning Common
```

Comment:
just remove cl21 altogether, don't mention it.

Reply:
Sure? In case there are external links pointing to it, it's friendly to not "break google".

I removed it from the index, so newcomers shouldn't see it.